lib/airbrake/rack/user.rb

```diff
@@ -22,12 +22,16 @@ def self.extract(rack_env)
         new(user) if user
       end
 
+      # rubocop:disable Lint/RescueException
       def self.try_current_user(rack_env)
         controller = rack_env['action_controller.instance']
         return unless controller.respond_to?(:current_user, true)
         return unless [-1, 0].include?(controller.method(:current_user).arity)
         controller.__send__(:current_user)
+      rescue Exception => _e
```

Comment:
We probably want to wrap line 30 only, not the whole method. This way we don't accidentally catch something from our own code.